projects/RabbitMQ.Client/client/impl/OutgoingCommand.cs

```diff
@@ -66,7 +66,7 @@ internal void Transmit(ushort channelNumber, Connection connection)
             var size = GetMaxSize(maxBodyPayloadBytes);
 
             // Will be returned by SocketFrameWriter.WriteLoop
```

Comment:
SocketFrameHandler ?

Reply:
Yup, the comment in code remains unchanged.

Reply:
Fixed